app/core/resources/constants/service.py

```diff
@@ -8,10 +8,32 @@
 # SERVICE CONFIG
 _service_section_name: str = "service"
 NAME: str = read_config(section=_service_section_name, value="name")
-TIMEOUT: int = int(read_config(section=_service_section_name, value="timeout"))
+TIMEOUT: int = int(
+    read_config(section=_service_section_name, value="timeout_in_seconds")
+)
 IP: str = read_config(section=_service_section_name, value="ip")
 PORT: int = int(read_config(section=_service_section_name, value="port"))
 
+ENABLE_DOCUMENT_THUMBNAIL: bool = (
+    True
+    if read_config(
+        section=_service_section_name, value="enable_document_thumbnail"
```

Comment:
Maybe we can pass also the default value (the third argument of the read_config method) so if, for some reason, the sysadmin delete the attribute in the config the application starts anyway